buildbot.mariadb.org/master.cfg

```diff
@@ -2767,7 +2767,7 @@ f_deb_autobake.addStep(steps.Trigger(name='galera-sst-rsync', schedulerNames=['s
     set_properties={"tarbuildnum" : Property("tarbuildnum"), "mariadb_version" : Property("mariadb_version"), "master_branch" : Property("master_branch"), "parentbuildername": Property("buildername"), "sst_mode": "rsync"}, doStepIf=lambda step: hasInstall(step) and savePackage(step) and hasFiles(step)))
 f_deb_autobake.addStep(steps.Trigger(name='major-minor-upgrade', schedulerNames=['s_upgrade'], waitForFinish=True, updateSourceStamp=False,
     set_properties={"tarbuildnum" : Property("tarbuildnum"), "mariadb_version" : Property("mariadb_version"), "master_branch" : Property("master_branch"), "parentbuildername": Property("buildername")}, doStepIf=lambda step: hasUpgrade(step) and savePackage(step) and hasFiles(step)))
-f_deb_autobake.addStep(steps.Trigger(name='dockerlibrary', schedulerNames=['s_dockerlibrary'], waitForFinish=False, updateSourceStamp=False,
+f_deb_autobake.addStep(steps.Trigger(name='dockerlibrary', schedulerNames=['s_dockerlibrary'], waitForFinish=True, updateSourceStamp=False,
```

Comment:
Any reason for this waitForFinish=True? It is needed for the build that triggers the dockerlibrary build to wait for it to finish?

Reply:
oh, totally misread this docs. Obviously wrong.

was hoping for the the dockerlibrary build to wait for the 3(4 with s390x) architecture deb_autobakes to finish so that 7ada134a2d51c88afbbffe642114e54641f57817 will have all the artifacts ready. So `hasDockerLibrary` needs changing, but I don't know how to do the waits on the `f_dockerlibrary`